netlify/functions/transDim.js

```diff
@@ -0,0 +1,167 @@
+require("dotenv").config();
+const nodemailer = require("nodemailer");
+const transDimConfig = require("../transDimConfig");
+
+// Function wrapper for sending email that returns a promise
+function sendEmail(mailOptions) {
+   console.log(mailOptions);
+   // Create the transporter for Nodemailer to post to the SMTP server
+   const transporter = nodemailer.createTransport({
+      host: transDimConfig.smtp.host,
+      port: transDimConfig.smtp.port,
+      secure: transDimConfig.smtp.secure,
+      requireTLS: transDimConfig.smtp.require_tls,
+
+      auth: {
+         user: transDimConfig.smtp.auth_user,
+         pass: transDimConfig.smtp.auth_pass
+      }
+   });
+   return transporter.sendMail(mailOptions);
+}
+
+exports.handler = async function (event, context) {
+   // Handle responses to OPTIONS preflight requests
+   if (event.httpMethod === "OPTIONS") {
+      console.log(event);
+      return {
+         statusCode: 200,
+         headers: {
+            "Access-Control-Allow-Origin": transDimConfig.allow_domain,
+            "Access-Control-Allow-Methods": "POST",
+            "Access-Control-Allow-Headers":
+               "Origin, X-Requested-With, Content-Type, Accept"
+         }
+      };
+   }
+   // Handle responses to POST requests
+   else {
+      console.log(event);
+
+      // Decode the JSON from the request body string
+      const bodyParsed = JSON.parse(event.body);
+      const { formData } = bodyParsed;
+
+      const enquirerName = formData.name;
+      const enquirerEmail = formData.email;
+      const enquirerMessage = formData.message;
+
+      const fields = [
+         { label: "Name", value: enquirerName },
+         { label: "Email", value: enquirerEmail },
+         { label: "Phone number", value: formData.phone },
+         { label: "Job title", value: formData.job },
+         { label: "Organisation", value: formData.organisation },
+         { label: "Postcode", value: formData.postcode },
+         {
+            label: "I'd like a ring back",
+            value: formData.ringBack == true ? "Yes" : "No"
+         },
+         {
+            label: "I'd like more information",
+            value: formData.moreInfo == true ? "Yes" : "No"
+         }
+      ];
+
+      // Deconstruct the recipients and styling from the config for ease of use
+      const emailRecipients = transDimConfig.email_recipients;
+      const emailStyling = transDimConfig.email_styling;
+
+      // Construct the email from the request body
+      const email = {
+         priority: "normal",
+         from: {
+            name: enquirerName,
+            address: process.env.EMAIL_FROM // the EnquiryWitch email to send from
+         },
+         replyTo: {
+            name: enquirerName, // 'reply to' name details
+            address: enquirerEmail // 'reply to' email address
+         },
+         to: emailRecipients.info,
```

Comment:
Just double checking `.info` is the value we want here - name seems a mismatch & odd not to comment it while the more obvious matches are annotated above.